go.mod

```diff
@@ -18,6 +20,6 @@ require (
 	go.starlark.net v0.0.0-20200821142938-949cc6f4b097
 	golang.org/x/arch v0.0.0-20190927153633-4e8777c89be4
 	golang.org/x/sys v0.0.0-20211117180635-dee7805ff2e1
-	golang.org/x/tools v0.1.8
+	golang.org/x/tools v0.1.8-0.20211028023602-8de2a7fd1736
```

Comment:
This actually downgraded x/tools to an older version which makes TestGeneratedDocs fail. I think also upgrading to 0.1.9 would be fine (or possibly pereferable).

Reply:
Done.